examples/llm/yamls/evaluation/gpt_neo_eval.yaml

```diff
@@ -0,0 +1,38 @@
+# Tokenizer
```

Comment:
Could we move both YAMLs here to a folder like `icl_eval/yamls`, to colocate the yamls with the `evaluate_model.py` script?